src/webui/www/private/views/filters.html

```diff
@@ -14,6 +14,7 @@
         <li id="stalled_filter"><a href="#" onclick="setFilter('stalled');return false;"><img src="icons/filterstalled.svg" alt="Stalled" />QBT_TR(Stalled (0))QBT_TR[CONTEXT=StatusFilterWidget]</a></li>
         <li id="stalled_uploading_filter"><a href="#" onclick="setFilter('stalled_uploading');return false;"><img src="icons/stalledUP.svg" alt="Stalled Uploading" />QBT_TR(Stalled Uploading (0))QBT_TR[CONTEXT=StatusFilterWidget]</a></li>
         <li id="stalled_downloading_filter"><a href="#" onclick="setFilter('stalled_downloading');return false;"><img src="icons/stalledDL.svg" alt="Stalled Downloading" />QBT_TR(Stalled Downloading (0))QBT_TR[CONTEXT=StatusFilterWidget]</a></li>
+        <li id="checking_filter"><a href="#" onclick="setFilter('checking');return false;"><img src="icons/checking.svg" alt="Checking" />QBT_TR(Checking (0))QBT_TR[CONTEXT=StatusFilterWidget]</a></li>
```

Comment:
For new code, the id should use camel case.

Reply:
I would suggest adding a whitespace after the first function call: `onclick="setFilter('checking'); return false;"`

Reply:
> For new code, the id should use camel case.

[updateFilter](https://github.com/AbeniMatteo/qBittorrent/blob/9ef4d5df1daea04501bbedd39d01e8deb638dc1e/src/webui/www/private/scripts/client.js#L371-L389) requires the id to use snake_case.

> I would suggest adding a whitespace after the first function call: `onclick="setFilter('checking'); return false;"`

Done